tools/flasher/flash.sh

```diff
@@ -134,7 +134,7 @@ cyan_underlined "Registering device information (5/5)"
 DEVINFO=`mpremote connect "port:$ESPTOOL_PORT" run ./register-device.py | sed "s/\r$//"`
 echo "$DEVINFO,$FIRMWARE_FILE_NAME,$AIRGUARD_VERSION" >> devices-flashed.txt
 
-esptool.py run
+mpremote reset
```

Comment:
Can we please add the ‘ connect "port:$ESPTOOL_PORT"’ part for this similar to the other mpremote calls to help on systems with multiple UART devices?

Reply:
yes, but obv outside of scope of this PR

Reply:
The current change as is breaks the tool on all systems with multiple UART devices. We would need to add that connection config for it to continue working.

Reply:
oh! oh misread! I see what you mean now! will check for that

Reply:
fixed! thank you for pointing this out!